appconfiguration/azure-spring-cloud-feature-management-web/azure-spring-cloud-feature-management-web-sample/README.md

```diff
@@ -1,6 +1,6 @@
 # Feature Management Web Sample shared library for Java
 
-This sample describes how to use [spring-cloud-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/appconfiguration/spring-cloud-azure-feature-management/README.md) to manage features and how to get configurations from App Configuration Service to Spring Environment.
+This sample describes how to use [azure-spring-cloud-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-cloud-feature-management-web_2.7.0/sdk/appconfiguration/azure-spring-cloud-feature-management/README.md) to manage features and how to get configurations from App Configuration Service to Spring Environment.
```

Comment:
Use `azure-spring-cloud-feature-management_2.7.0` instead of `azure-spring-cloud-feature-management-web_2.7.0`.